app/(main)/generate/GeneratePolicyTabContent.tsx

```diff
@@ -17,7 +17,7 @@ export default function GeneratePolicyTabContent({
       <div className="py-4 md:py-0" />
       <div className="grid grid-flow-row px-2 md:w-full md:max-w-[800px]">
         <h2 className="text-lg font-bold leading-normal text-[#4A558E]">
-          Step 3: Generate your AI Policy
+          Step 2: Generate your AI Policy
```

Comment:
this needs to be a conditional change. If you go to `www.coursepolicy.ai/generate`, it has three steps instead of two.